templates/registry_list.php

```diff
@@ -4,7 +4,12 @@
 $this->data['head']  = '<link rel="stylesheet" type="text/css" href="/' . $this->data['baseurlpath'] . 'module.php/oauth2/resources/style.css" />' . "\n";
 $this->includeAtTemplateBase('includes/header.php');
 
-$moduleurlpath = '/' . $this->data['baseurlpath'].'/module.php/oauth2/';
```

Comment:
Yes, this is a bug, but because there are a extra '/' before `module.php`. This should be

```
$moduleurlpath = '/' . $this->data['baseurlpath']. 'module.php/oauth2/';
```

That's all.